h2o-py/tests/testdir_pipeline/pyunit_scale_pca_rf.py

```diff
@@ -21,10 +18,50 @@ def scale_pca_rf_pipe():
                    ("pca", H2OPCA(k=2)),
                    ("rf", H2ORandomForestEstimator(seed=42,ntrees=50))])
   pipe.fit(iris[:4],iris[4])
+  print(pipe)
+
+
+def scale_pca_rf_pipe_new_import():
+
+  from h2o.transforms.preprocessing import H2OScaler
+  from h2o.estimators.pca import H2OPrincipalComponentAnalysisEstimator as H2OPCA
```

Comment:
I think the alias "H2OPCA" makes the test a little bit harder to read (because there already is an H2OPCA class)

Reply:
Ok, I will change it.